src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php

```diff
@@ -227,7 +227,9 @@ private function generateMetadataFields(ManageEntity $entity): array
         if ($entity->getMetaData()->getLogo() instanceof Logo && $entity->getMetaData()->getLogo()->getUrl() !== '') {
             $metadata = array_merge($metadata, $this->generateLogoMetadata($entity));
         }
-
+        if ($entity->getMetaData()?->getCoin()->isIdpVisibleOnly() !== null) {
```

Comment:
What does this mean? Tis is a fairly complex equation, and it is not clear what is meant. Using a descriptive method could maybe clarify a bit

Reply:
What part of the equation is complex in your opinion? I find it rather expressive, but I'm kind of biassed ;P

Reply:
The whole part... I have no idea what true and false mean in this case